### PR TITLE
feat: update charts for optimus 0.11.1

### PR DESCRIPTION
**Pull request by sbchaos**

### CODE REVIEW

File: stable/optimus/values.yaml [hunk 1]
```diff
@@ -84,11 +89,15 @@ service:
...
       name: http
+    - port: 8081
+      name: grpc
+      targetPort: grpc
+      protocol: TCP
   annotations:
-    projectcontour.io/upstream-protocol.h2c: 80,http
+    projectcontour.io/upstream-protocol.h2c: grpc
 
 ingress:
```
Comment (deryrahman) [inline stable/optimus/values.yaml:102]: could you add `additionalIngress` related fields on values.yaml?

File: stable/optimus/templates/ingress.yaml [hunk 2]
```diff
@@ -1,28 +1,18 @@
...
   labels:
-    {{- include "app.labels" . | nindent 4 }}
+    {{- $labels | nindent 4 }}
   {{- with .Values.ingress.annotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
+  {{- if .Values.ingress.className }}
```
Comment (deryrahman) [inline stable/optimus/templates/ingress.yaml:15]: please also add className field under ingress on value.yaml

File: stable/optimus/values.yaml (hunk 1 above)
Comment (deryrahman) [reply stable/optimus/values.yaml:102]: please refer to this https://github.com/goto/charts/blob/main/stable/app/values.yaml

**Review by deryrahman — CHANGES_REQUESTED** [review]

File: stable/optimus/templates/ingress.yaml [hunk 3]
```diff
@@ -1,28 +1,18 @@
...
   ingressClassName: {{ .Values.ingress.className }}
   {{- end }}
   {{- if .Values.ingress.tls }}
```
Comment (deryrahman) [inline stable/optimus/templates/ingress.yaml:18]: please remove this, if it's not needed, as we don't specify it on values.yaml

File: stable/optimus/templates/ingress.yaml [hunk 4]
```diff
@@ -43,19 +33,56 @@ spec:
...
+    {{- $labels | nindent 4 }}
+  {{- with .annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .className }}
+  ingressClassName: {{ .className }}
+  {{- end }}
+  {{- if .tls }}
```
Comment (deryrahman) [inline stable/optimus/templates/ingress.yaml:61]: also this one, can be removed if it's not needed

**Review by deryrahman — APPROVED** [review]
LGTM after the feedbacks are addressed

File: stable/optimus/templates/ingress.yaml (hunk 3 above)
Comment (sbchaos) [reply stable/optimus/templates/ingress.yaml:18]: Values.yaml is only indicative, the feature might be used in production.

**Pull request merged**